projekt_do_pracy_na_labach/fiszki_backend/Dockerfile

```diff
@@ -1,2 +1,44 @@
-FROM python:3.10
+# zadanie na 3.0
+FROM python:3.10  
```

Comment:
Możesz też po prostu wrzucić WORKDIR bez konieczności użycia RUN (choć doceniam za pomysł 👍 ). Jak by to wyglądało po zmianach
```
FROM python:3.10


WORKDIR /app 

COPY . .

RUN pip install -r requirements.txt 

CMD ["python", "main.py"]
```